src/client/contract-client.js

```diff
@@ -217,10 +212,11 @@ class Client {
  * Returns an XDR type based on the type defined in the abiJson.
  *
  * @param output The string defined in abijson for the contract type.
+ * @param output The string defined in abijson for the contract type.
```

Comment:
Looks like you forgot the update the second param type description here.

Reply:
Fixed, thanks.